src/components/categoryFilter/CategoryFilter.jsx

```diff
@@ -0,0 +1,22 @@
+import { Title, List, ListItem, Name, Count } from './styles';
+import CATEGORIES from 'constants/CATEGORIES';
```

Comment:
название файлов не стоит в верхнем регистре писать. Поменяй названия файлов с константами на lower camel case

Reply:
Изменил названия файлов с константами на lower camel case:
`CATEGORIES.js` → `categories.js` 
`SIZES.js` → `sizes.js` 
`SORT_TYPES.js` → `sortTypes.js` 
```suggestion
import CATEGORIES from 'constants/categories';
```